File: src/main/java/eu/mjindra/utils/dices/D20.java
package eu.mjindra.utils.dices;

/**
 * A D20 die.
 *
 * @author Martin Jindra
 * @version 31.05.2023
 */
public class D20 extends Dice {

    public D20(byte amount) {
        this(amount, (byte) 20);
    }

    public D20(byte amount, byte modifier) {
        super(amount, (byte) 20, modifier);
    }

}

File: lib/dnd/dices/d20.dart
import 'package:helios/dnd/dices/dice.dart';
import 'package:helios/dnd/dices/die.dart' show Die;

/// A D20 die.
class D20 extends Die {
  D20() : super(DiceSides.d20);

  D20.mod(int modifier) : super.mod(DiceSides.d20, modifier);
}
